scipy/stats/_hypotests.py

```diff
@@ -756,6 +756,9 @@ def barnard_exact(table, alternative="two-sided", pooled=True, n=32):
                 \frac{1}{c_2})
             }
         }
+    with :math:`\hat{p_1}, \hat{p_2}` and :math:`\hat{p}` the estimator of
```

Comment:
```suggestion

    with :math:`\hat{p_1}, \hat{p_2}` and :math:`\hat{p}` the estimator of
```
I think this is causing your CI failure:
```
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/stats/_hypotests.py:docstring of scipy.stats._hypotests.barnard_exact:88: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

Reply:
I am ok with this :-)